### PR TITLE
Option to override table prefix & resolve size monitor type change

### PR DESCRIPTION
**Pull request by bheltzel**
Implemented an option to override the table prefix for PROD_INSIGHTS since customers don't have the full prefix.

### CODE REVIEW

File: marketplace.json [hunk 1]
```diff
@@ -5,14 +5,17 @@
...
+
     "constants": {
       "CONNECTION_NAME": {
         "label": "Looker Connection Name",
         "value_constraint": "connection"
       },
       "DATASET_NAME": {
         "label": "Monte Carlo Dataset Name"
+      },
+      "DATASET_NAME": {
```
Comment (mdediana) [inline marketplace.json]: Should this be `TABLE_PREFIX`?

File: views/insight_monitors.view.lkml [hunk 2]
```diff
@@ -126,14 +126,25 @@ view: insight_monitors {
 
   dimension: size_monitor {
     type: yesno
-    sql: ${TABLE}."SIZE_MONITOR" ;;
+    # sql: ${TABLE}."SIZE_MONITOR" ;;
+    sql: ${volume_change_monitor} or ${unchanged_size_monitor} ;;
```
Comment (mdediana) [inline views/insight_monitors.view.lkml:130]: It's unrelated to the prefix change right?

**Review by glencwillis — APPROVED** [review]

**Review by mdediana — APPROVED** [review]
Looks good, but see comments.

File: marketplace.json (hunk 1 above)
Comment (bheltzel) [reply marketplace.json]: Amazing! Found a bug in something you didn't even know! I fixed it.

File: views/insight_monitors.view.lkml (hunk 2 above)
Comment (bheltzel) [reply views/insight_monitors.view.lkml:130]: Yes, this just happened to be the bug from changing our size monitors into a separate report.

**Pull request merged**